src/coreclr/vm/jitinterface.cpp

```diff
@@ -3104,12 +3099,6 @@ void CEEInfo::ComputeRuntimeLookupForSharedGenericToken(DictionaryEntryKind entr
                 IfFailThrow(sigptr.GetData(&data));
                 pResult->offsets[2] = sizeof(TypeHandle) * data;
 
-                if (MethodTable::IsPerInstInfoRelative())
-                {
-                    pResult->indirectFirstOffset = 1;
-                    pResult->indirectSecondOffset = 1;
```

Comment:
Are `indirectFirstOffset` and `indirectSecondOffset` always 0 now / could they be removed in the future?

Reply:
No, they may still be needed for cross-dll handling in NativeAOT in the future. So, its not clear its safe to remove them until we've landed on an exact design for that.